pkg/services/featuremgmt/toggles_gen_test.go

```diff
@@ -41,6 +41,61 @@ func TestFeatureToggleFiles(t *testing.T) {
 		}
 	})
 
+	ownerlessFeatures := map[string]bool{
+		"alertingBigTransactions":           true,
+		"trimDefaults":                      true,
+		"disableEnvelopeEncryption":         true,
+		"database_metrics":                  true,
+		"prometheusAzureOverrideAudience":   true,
+		"publicDashboards":                  true,
```

Comment:
isn't this one and the next one `grafanaDashboardsSquad` @natellium?

Reply:
Yes, this one and `publicDashboardsEmailSharing`